### ISSUE TITLE
assertion failure trying to export bundle

### ISSUE THREAD

**Issue by cabbey** [sequelpro/sequelpro#2261]
Steps to reproduce:
1. attempt to export a custom bundle from the bundle editor. 

Expected Behavior: file foo.spBundle created

Actual Behavior: console reports an assertion failure:
```
10/2/15 2:54:28.914 PM Sequel Pro[14478]: assertion failed: 14F27: libxpc.dylib + 62447 [5C829202-962E-3744-8B50-00D38CC88E84]: 0x89
```

Additional Notes:
The bundle I'm attempting to export is just a basic copy bundle, derived from CopyAsJSON that I downloaded from the bundle collection.

**Comment by dmoagx** [comment]
Can reproduce with 1.1 on 10.9, 10.11
10.9 doesn't log an assertion failure, but export doesn't work either.

**Comment by cabbey** [comment]
Ah yeah, I forgot to mention that this is sequel pro 1.1 (build 4499) running on OS/X 10.10.5.

**Comment by dmoagx** [comment]
Until 1.1.1 is released you can either downgrade to 1.0.2 (http://www.sequelpro.com/release-archive) or use a future experimental build including the fix: http://nightly.sequelpro.com/

**Comment by dmoagx** [comment]
Issue fixed in 1.1.1
See http://www.sequelpro.com/blog/2016.02/sequel-pro-1-1-1/
